drivers/tty/serdev/Makefile

```diff
@@ -5,3 +5,5 @@ obj-$(CONFIG_SERIAL_DEV_BUS) += serdev.o
 
 obj-$(CONFIG_SERIAL_DEV_CTRL_TTYPORT) += serdev-ttyport.o
 obj-$(CONFIG_SERIAL_DEV_MUX_PIBRIDGE) += pibridge.o
+
+CFLAGS_pibridge.o = -I$(src)
```

Comment:
Are you sure this is still needed? I just did a successful compile test without this line.

Reply:
If I omit this I get the compiler error
./include/trace/define_trace.h:95:42: fatal error: ./pibridge_trace.h: Datei oder Verzeichnis nicht gefunden
so I guess its still needed

Reply:
That is really odd, I cannot reproduce the error. I've compile-tested with revpi-v7_defconfig and just removed this line and it compiled cleanly.

Reply:
This is the only remaining open question, otherwise I'm fine with this pull request.

Could you explain the steps to get that error message? I've done another compile test and again did not get the error.

Reply:
According to this LWN article
https://lwn.net/Articles/383362/
all modules have to include the path of custom tracepoint definitions in EXTRA_CFLAGS or the source code file specific CFLAGS.
You can see examples for the latter in the kernel code, e.g.
https://elixir.bootlin.com/linux/latest/source/drivers/base/regmap/Makefile

Are you sure you have tracing enabled in the kernel?
How did you build you kernel config?

Reply:
I used revpi-v7_defconfig. I was under the impression that tracing is enabled with that config.

Reply:
I made a clean new build with the current revpi-v7_defconfig, set CONFIG_SERIAL_DEV_MUX_PIBRIDGE=y and if I omit the CFLAGS_pibridge.o = -I$(src) I get the compiler error.